src/components/serverPaginatedTable/serverPaginatedTable.js

```diff
@@ -40,9 +39,14 @@ class ServerPaginatedTableView extends React.Component {
 
   getSrcData = () => this.props.data;
 
+  rowsSelectedTrigger = (displayedData) => this.props.options.rowsSelectedTrigger(
```

Comment:
Can you add an edge case where we don't need rowsselectedTrigger(Which will benefit the cart)